pkg/ostree/ostree.go

```diff
@@ -25,8 +25,9 @@ var (
 // SourceSpec serves as input for ResolveParams, and contains all necessary
 // variables to resolve a ref, which can then be turned into a CommitSpec.
 type SourceSpec struct {
-	URL  string
-	Ref  string
+	URL string
+	Ref string
+	// RHSM is deprecated, use Resolve function with MTLS configuration
```

Comment:
Could the mTLS configuration be added to the SourceSpec?  I imagine that would make things simpler, but I haven't written it out to be sure.

Reply:
Yeah well I thought that `SourceSpec` is a type I should not touch. Now that I think about it, I can add fields directly in there that is more simple. Let me try.